public/assets/css/main.css

```diff
@@ -45,3 +57,11 @@ p {
 .cta:hover, .cta:focus {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
+
+.screenshot {
+    display: block;
+    margin: 2em auto;
+    max-width: 100%;
+    max-height: 30em;
```

Comment:
Are you sure you also need to specify a `max-height` if you've already specified a `max-width`?